nix/nixos/cardano-graphql-service.nix

```diff
@@ -106,6 +97,11 @@ in {
       requires = [ "graphql-engine.service" ];
       environment = {
         HASURA_URI = hasuraBaseUri;
+        #POSTGRES_HOST =
```

Comment:
@disassembler @craigem 
These ENVs need to be set. Storing the secrets in files means the chance of leaking via command arguments or ENVs is removed.